wire-gradle-plugin/src/test/projects/java-project-java-protos/build.gradle

```diff
@@ -6,10 +6,10 @@ plugins {
 mainClassName = 'com.squareup.dinosaurs.Sample'
 
 repositories {
+  mavenLocal()
   mavenCentral()
 }
 
 dependencies {
-  implementation 'com.squareup.wire:wire-runtime:+'
```

Comment:
Somehow `+` still resolves to `3.0.0-alpha01`, is there a way to make it resolve to the latest SNAPSHOT?